lms/templates/courseware/program_marketing.html

```diff
@@ -1,3 +1,424 @@
+                        <a href="#instructor-details-mobile-${index}" class="instructor-label">${instructor.get('name')}</a>
+                      </div>
+                      % if instructor.get('position'):
+                        <div class="dept">${instructor['position'].get('organization_name')}</div>
+                      % endif
+                    </div>
+                    <div class="modal modal-custom" id="instructor-details-mobile-${index}">
+                      <a href="#" class="focusKeeper"></a>
+                      <div class="btn-close modal_close"><a href="#"><i class="fa fa-close"></i></a></div>
+                      <div class="modal-body">
+                        <div class="modal-header">
+                          <div class="instructor-data">
+                            <div class="thumbnail">
+                              <img src="${instructor['image']}" alt="${instructor.get('name')}">
+                            </div>
+                            <h3>
+                              <span class="instructor-name">${instructor.get('name')}</span>
+                              % if instructor.get('position'):
+                                <span>
+                                  ${Text(_('{position} at {organization}')).format(
+                                    position=instructor['position'].get('title'),
+                                    organization=instructor['position'].get('organization_name')
+                                  )}
+                                </span>
+                              % endif
+                            </h3>
+                          </div>
+                        </div>
+                        <div class="instructor-bio">${instructor['bio']}</div>
+                      </div>
+                      <a href="#" class="focusKeeper"></a>
+                    </div>
+                  </div>
+                %endfor
+              </div>
+            </div>
+          </div>
+
+          % try:
+            <%include file="_${program_type_slug}_faq.html" args="program_type=program_type" />
+          % except exceptions.TemplateLookupException:
+            ## pass
+          % endtry
+
+          <div class="accordion-item">
+            <div class="accordion-head">${_("FAQ")}</div>
+            <div class="accordion-content">
+              <ul class="list-unstyled faq-links-list">
+                % for item in faq:
+                  <li class="item">
+                    <a href="#!">${item['question']}</a>
+                    <div class="answer hidden">${item['answer']}</div>
+                  </li>
+                % endfor
+              </ul>
+            </div>
+          </div>
+        </div>
+        <blockquote class="program-quote visible-sm">
+          <div class="quote">
+            % if endorsement:
+              <span>"${endorsement_quote}"</span>
+                % if endorser and endorsement_image:
```

Comment:
@marjev I think the endorsement_image check should only be for the img, and not for name, position etc, just like you did in desktop view below.